opensearch_py_ml/ml_commons/cli/connector_create.py

```diff
@@ -0,0 +1,364 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+from colorama import Fore, Style, init
+from rich.console import Console
+
+from opensearch_py_ml.ml_commons.cli.AIConnectorHelper import AIConnectorHelper
+from opensearch_py_ml.ml_commons.cli.connector_base import ConnectorBase
+from opensearch_py_ml.ml_commons.cli.ml_models.AlephAlphaModel import AlephAlphaModel
+from opensearch_py_ml.ml_commons.cli.ml_models.BedrockModel import BedrockModel
+from opensearch_py_ml.ml_commons.cli.ml_models.CohereModel import CohereModel
+from opensearch_py_ml.ml_commons.cli.ml_models.DeepSeekModel import DeepSeekModel
+from opensearch_py_ml.ml_commons.cli.ml_models.OpenAIModel import OpenAIModel
+from opensearch_py_ml.ml_commons.cli.ml_models.SageMakerModel import SageMakerModel
+
+# Initialize Rich console for enhanced CLI outputs
+console = Console()
+
+# Initialize colorama for colored terminal output
+init(autoreset=True)
+
+
+class Create(ConnectorBase):
```

Comment:
This name sounds not good. It's a verb. 
I think we can extend this class for also supporting updating/deleting connector.
Some suggestions:
`ConnectorManager` , `ConnectorHandler`